lib/utimes.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var fs = require('graceful-fs');
+
+function utimes(writePath, stat, cb) {
+  if (stat.mtime) {
+    stat.atime = stat.atime || new Date();
```

Comment:
Ok! I didn't get it before, good point. Fixed and added tests, although they are failing because someone is adding an undefined `mode`... :-/ I've done it on a different commit, so it could be ignored on this pull-request if needed and cherry-picked on another branch to search who is adding the `mode`.